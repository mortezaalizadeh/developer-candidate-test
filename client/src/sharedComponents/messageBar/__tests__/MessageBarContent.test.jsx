import React from 'react';
import renderer from 'react-test-renderer';
import MessageBarContent from '../MessageBarContent';

it('should render correctly', () => {
  const tree = renderer.create(<MessageBarContent variant="error" message="This is a test error message" />).toJSON();

  expect(tree).toMatchSnapshot();
});
