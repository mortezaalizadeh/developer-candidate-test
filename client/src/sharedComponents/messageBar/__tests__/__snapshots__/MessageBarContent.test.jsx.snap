// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  aria-describedby="client-snackbar"
  className="MuiTypography-root-38 MuiTypography-body1-47 MuiPaper-root-11 MuiPaper-elevation6-19 MuiSnackbarContent-root-8 MessageBarContent-error-2"
  role="alertdialog"
>
  <div
    className="MuiSnackbarContent-message-9"
  >
    <span
      className="MessageBarContent-message-7"
      id="client-snackbar"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-74 MessageBarContent-icon-5 MessageBarContent-iconVariant-6"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
        />
      </svg>
      This is a test error message
    </span>
  </div>
  <div
    className="MuiSnackbarContent-action-10"
  >
    <button
      aria-label="Close"
      className="MuiButtonBase-root-89 MuiIconButton-root-83 MuiIconButton-colorInherit-84"
      disabled={false}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiIconButton-label-88"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-74 MessageBarContent-icon-5"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
          />
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
        </svg>
      </span>
      <span
        className="MuiTouchRipple-root-92"
      />
    </button>
  </div>
</div>
`;
